server/src/main/java/org/opensearch/node/Node.java

```diff
@@ -32,46 +32,40 @@
 
 package org.opensearch.node;
 
+import java.io.BufferedWriter;
```

Comment:
Could you please revert the package arrangement?

Reply:
I can but IMO we should enforce a standard ordering. I believe package -> java -> other -> static is pretty common.

Reply:
I mean is that spotless related? Or it is from the IDE?